src/modules/75-cf/pages/home/trialPage/CFTrialPanel.tsx

```diff
@@ -1,43 +1,30 @@
 /*
- * Copyright 2021 Harness Inc. All rights reserved.
+ * Copyright 2022 Harness Inc. All rights reserved.
  * Use of this source code is governed by the PolyForm Shield 1.0.0 license
  * that can be found in the licenses directory at the root of this repository, also available at
  * https://polyformproject.org/wp-content/uploads/2020/06/PolyForm-Shield-1.0.0.txt.
  */
 
-import React, { FC, ReactNode } from 'react'
-import { Heading, Layout, Text, Container, Button, useToaster, ButtonVariation } from '@harness/uicore'
 import { Color, FontVariation } from '@harness/design-system'
+import { Button, ButtonVariation, Heading, Layout, Text, useToaster } from '@harness/uicore'
+import type { FC, ReactNode } from 'react'
+import React from 'react'
```

Comment:
You can combine these
```suggestion
import React, { FC, ReactNode } from 'react'
```

Reply:
Thanks, done